frontend/server/src/Controllers/User.php

```diff
@@ -167,6 +167,42 @@ public static function createUser(
             'verification_id' => \OmegaUp\SecurityTools::randomString(50),
             'is_private' => boolval($createUserParams->isPrivate),
         ];
+        \OmegaUp\Validators::validateNumber(
+            $createUserParams->birthDate,
+            'birth_date'
+        );
+        if ($createUserParams->birthDate >= strtotime('-13 year', \OmegaUp\Time::get())) {
+            // Fill all the columns refering to user's parent
+            $userData['parental_verification_token'] = \OmegaUp\SecurityTools::randomHexString(
+                25
+            );
+            $userData['creation_timestamp'] =  \OmegaUp\Time::get();
+            $userData['parent_email_verification_initial'] = \OmegaUp\Time::get();
+            $userData['parent_email_verification_deadline'] = strtotime('+7 days', \OmegaUp\Time::get());
+              
+            $user = \OmegaUp\DAO\Users::getAll();
+            $email = \OmegaUp\DAO\Emails::getByPK($user->parentEmail);
+
+            if (!self::$sendEmailOnVerify) {
+                self::$log->info(
+                    'Not sending email beacause sendEmailOnVerify = FALSE'
+                );
+                return;
+            }
+    
+            $subject = \OmegaUp\Translations::getInstance()->get(
+                'parentEmailSubject'
+            );
+            $body = \OmegaUp\ApiUtils::formatString(
+                \OmegaUp\Translations::getInstance()->get('parentEmailBody'),
+                [
+                    'parental_verification_token' => strval($user->parental_verification_token),
+                ]
+            );
+    
+            \OmegaUp\Email::sendEmail([$parentEmail], $subject, $body);
```

Comment:
You could get the parent's email directly from the UserParams class:
```suggestion
            \OmegaUp\Email::sendEmail([$createUserParams->parentEmail], $subject, $body);
```
So, you can add thisnew field such as `birthDate` field was added